contributing/DEVELOPING.md

```diff
@@ -84,6 +84,22 @@ In particular, we try to maintain "it might not be perfect but isn't broken"-sup
 
 We also recommend scanning the [spec](../spec/). The non-element part should help understand some of the design aspects.
 
+## Code Reviewes
```

Comment:
s/Code Reviewes/Code Reviews

Reply:
I was just trying to help with GFI stuff Naina ;) fixed